src/Covid19Dashboard/PlotChart/index.jsx

```diff
@@ -210,20 +224,30 @@ class PlotChart extends PureComponent { // eslint-disable-line react/no-multi-co
       />
       <XAxis
         dataKey='date'
-        tick={<PlotChartAxisTick axis='x' type='date' />}
+        tick={<PlotChartAxisTick
+          axis='x'
+          type='date'
+          labelMaxLength={axisLabelMaxLength}
```

Comment:
this is meaningless since the type is not `string`?

Reply:
it changes the font size for numbers and dates as well